lib/fiware-iotagent-lib.js

```diff
@@ -259,7 +259,9 @@ exports.setDataUpdateHandler = contextServer.setUpdateHandler;
 exports.setCommandHandler = contextServer.setCommandHandler;
 exports.setDataQueryHandler = contextServer.setQueryHandler;
 exports.setConfigurationHandler = contextServer.setConfigurationHandler;
+exports.setRemoveConfigurationHandler = contextServer.setRemoveConfigurationHandler;
 exports.setProvisioningHandler = contextServer.setProvisioningHandler;
+exports.setRemoveDeviceHandler = contextServer.setRemoveDeviceHandler;
 exports.setNotificationHandler = contextServer.setNotificationHandler;
```

Comment:
It would be great if the new functions could be documented along with the other ones in the Usermanual? 

E.g. https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/usermanual.md#iotagentlibsetconfigurationhandler

Reply:
Fixed with f4747f7b1b3d315a3e2ba3613d076fcd2b844d44